test/data-pipeline/emr-job-submitter.test.ts

```diff
@@ -388,4 +418,24 @@ describe('Data Process -- EMR Serverless job submitter function', () => {
     ]);
   });
 
+  test('Write 2 state files in S3', async () => {
+    lambdaMock.on(ListTagsCommand).resolves({ Tags: {} });
+    await EMRServerlessUtil.start({
+      startTimestamp: new Date(startTimestamp).getTime(),
+    }, context);
+    expect(putStringToS3Mock.mock.calls.length).toEqual(2);
+    expect(putStringToS3Mock.mock.calls[0][2]).toEqual('pipeline-prefix/job-info/test_proj_001/job-jobId007.json');
+    expect(putStringToS3Mock.mock.calls[1][2]).toEqual('pipeline-prefix/job-info/test_proj_001/job-latest.json');
+  });
+
+  test('Write 1 state file in S3 when event.reRunJob=true', async () => {
```

Comment:
How to submit the  job with `reRunJob=true`?

Reply:
if you submit a EMR job via lambda using below event: 
```json {
reRunJob: true,
startTimestamp: 123455656,
endTimestamp: 999999999
}```
lambda will not update the latest job state in s3, otherwise the next job will set startTimestamp to `999999999`

Reply:
So `reRunJob=true` only happens when manually triggering the re-run of a job?

Reply:
yes.